meet_balls_app/tests.py

```diff
@@ -64,6 +73,82 @@ def test_successful_logout(self, client, player):
         client.post(login_url, {'username': username, 'password': password})
         logout_url = reverse('logoutUser')
         response = client.post(logout_url)
+        assert response.status_code == 302
+        assert response.url == reverse('home')
+        assert response.wsgi_request.user.is_authenticated is False
+
+
+@pytest.mark.django_db
+class TestRegister:
+    def test_registerUser_valid(self, client):
```

Comment:
Shouldn't we test whether the player has been created and now exists?

Reply:
Agree. Fixed.